src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -29,8 +29,15 @@ make those filters work as expected.
 
 ## Extended data types and schema scopes
 
-There are various kinds of extended data. Listings support public data,
-private data, and public metadata. All these are editable in Console by
+There are various kinds of extended data. Search schema support differs depending on whether you
+use Marketplace API or Integration API and also on whether the `schema-for` value is `listing` or `user`. The below table summarizes what scopes of search schema is supported based on different factors.
+
+|          | Marketplace API                | Integration API                  |
+| -------- | ------------------------------ | -------------------------------- |
+| listing  | public, meta, private, keyword | public, meta, keyword (private, protected coming in the future) |
```

Comment:
TODO: If Marketplace API listings query allows private search schemas, API reference requires an update as well, since it only lists pub_* and meta_* as possible query params for [Marketplace API listing query](https://www.sharetribe.com/api-reference/marketplace.html#query-listings).

Reply:
Oh I think this has been my mistake :) will fix it.

Reply:
I wouldn't mention any future plans here, as they are uncertain and there's no point to over-promise something we might not do anytime soon.

Reply:
@bladealslayer 

I will delete the mention about future plans. But I know we have discussed this could come in future that's why I mentioned them here.

"If we know" that they are coming, I imagined a few good effects of mentioning planned features. 

* Existing customers can plan what to do with our future features. And "if they are planning to quit our subscription because we are missing X, Y and Z", we can reassure them they are coming soon they just need to wait some time.
* Even though some potential customers know that they are missing some features right now, we can let them know we have plans for them, and this might convince them to start using Flex now.
* If they read our docs, it can prevent questions like "Are you planning to add X, Y and Z features?" (well we can still get questions like "When are you planning to do X, Y and Z"...
* There might be more but more or less based on the same logic...

I am totally fine with not mentioning any planned features and deleting them now. This is just food for thoughts and we wanna consider the positive side of the trade offs as well.

Reply:
Yeah, indeed. However, if we don't really have concrete plans on building those, people may have the wrong expectation and think we've promised those features but aren't delivering them.